### DISCUSSION TITLE
Discussions of development

### DISCUSSION THREAD

**Discussion by WebFreak001** [WebFreak001/code-debug#349] [General]
@GitMensch @brownts I have enabled Discussions on this repository so we can rather discuss about development here than on issues that might be unrelated to what is being discussed.

**Comment by GitMensch** [comment]
Thank you, that is useful.

As a side note: I tried to use the milestones more to get a better overview about features and bugs that should land "somewhere". We have a current bug-fix milestone, a feature one and increased use of the backlog https://github.com/WebFreak001/code-debug/milestones.

**Comment by brownts** [comment]
I like the idea of the milestones, it helps to quantify what we expect to go into the next (or future version) of the extension.  It also helps us know when we should be ready to do a release.

**Comment by lavyadrian** [comment]
Hello! I just installed your extension to debug nim programs and a feature that I had come to rely on with vscode debugging has disappeared, namely the disassembly view, which still appears with C language programs...is there a way to make it appear again?
EDIT : I did not make this into an issue because I assume I am just missing the knowledge of how to do it.
